src/Features/CSharp/Portable/BraceCompletion/AbstractCurlyBraceOrBracketCompletionService.cs

```diff
@@ -120,7 +121,8 @@ private static bool ContainsOnlyWhitespace(SourceText text, int openingPosition,
 
             var rootToFormat = document.Root.ReplaceToken(closingToken, newClosingToken);
             annotatedNewline = rootToFormat.GetAnnotatedTrivia(s_closingBraceNewlineAnnotation).Single();
-            document = document.WithChangedRoot(rootToFormat, cancellationToken);
+
+            document = GetUpdatedDocument(document, new[] { new TextChange(closingToken.FullSpan, newClosingToken.ToFullString()) }, rootToFormat);
```

Comment:
```suggestion
            document = GetUpdatedDocument(document, [new TextChange(closingToken.FullSpan, newClosingToken.ToFullString())], rootToFormat);
```

Reply:
huh, IDE gave me an error when I tried that earlier, but it works now. IDE's been wonky all day for me though.